app/helpers/filter_dropdown_helper.rb

```diff
@@ -0,0 +1,89 @@
+module FilterDropdownHelper
+  def list_eq_filter_dropdown(attribute, collection, t_scope: nil)
+    set_local_q
+    t_scope ||= [:simple_form, :options, controller_name.singularize.to_sym]
+    @t_scope = { scope: t_scope.push(attribute) }
```

Comment:
Using an instance variable for this is a bit confusing, it would be better if you could pass the scope on to the other methods.